ingestion/src/metadata/data_insight/runner/run_result_registry.py

```diff
@@ -0,0 +1,164 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+function to compute kpi
+"""
+
+from __future__ import annotations
+
+import ast
+import traceback
+
+from metadata.generated.schema.dataInsight.kpi.basic import KpiResult, KpiTarget
+from metadata.generated.schema.dataInsight.type.percentageOfEntitiesWithDescriptionByType import (
+    PercentageOfEntitiesWithDescriptionByType,
+)
+from metadata.generated.schema.dataInsight.type.percentageOfEntitiesWithOwnerByType import (
+    PercentageOfEntitiesWithOwnerByType,
+)
+from metadata.generated.schema.type.basic import FullyQualifiedEntityName
+from metadata.utils.dispatch import enum_register
+from metadata.utils.logger import sqa_interface_registry_logger
+
+logger = sqa_interface_registry_logger()
+
+
+def percentage_of_entities_with_description_kpi_result(
+    kpi_target: list[KpiTarget],
+    results: list[PercentageOfEntitiesWithDescriptionByType],
+    kpi_fqn: FullyQualifiedEntityName,
+    timestamp: int,
+) -> KpiResult:
+    """Validate percentage of entities with description kpi
+
+    Args:
+        kpi_target (list[KpiTarget]):
+        results (list[PercentageOfEntitiesWithDescriptionByType]):
+
+    Raises:
+        ValueError:
+        RuntimeError:
+
+    Returns:
+        KpiResult:
+    """
+    if not results:
+        raise ValueError("Cannot compute KPI. No results found")
+
+    latest_chart_result = {}
+
+    print(results)
```

Comment:
ups

Reply:
pushed a change, sometimes I forget we can do it directly from the UI which is awesome